src/lib/common/Kconfig

```diff
@@ -88,18 +88,51 @@ config DEFAULT_LOG_LEVEL_DEBUG
 	bool "debug"
 endchoice
 
-config KDBUS
-	bool "Kdbus"
-	depends on SOL_PLATFORM_LINUX && HAVE_SYSTEMD && HAVE_KDBUS
+config SOL_BUS
+	bool "Sol-bus"
+	depends on SOL_PLATFORM_LINUX && HAVE_SYSTEMD && HAVE_SD_BUS
 	default n
 
+menu "Hardware Options"
+config USE_PWM
+	bool "PWM Support"
+	help
+          Enables PWM support.
```

Comment:
The help text isn't helping much. They're pretty similar to these comments, in fact:

    int i = 0; /* Declare an integer variable named "i", and initialize it to "0" */
    i++; /* Increment integer variable i */

Maybe expand the acronym in the help text ("Enable Pulse-Width Modulation support", "Enables Universal Asynchronous Receiver/Transmitter support", "Enable General Purpose I/O support", etc), give an example or two (e.g. for PWM, "Generally used to control the power supplied to devices connected to an I/O port, such as motors or LEDs."), and optionally a Wikipedia link (e.g. https://en.wikipedia.org/wiki/Pulse-width_modulation).